### DISCUSSION TITLE
Translation contribute

### DISCUSSION THREAD

**Discussion by Dolland** [velitasali/gnome-shell-extension-awesome-tiles#5] [Q&A] [answered]
Hi Veli! 
I've done a italian translation .po but i'm wondering how to merge this file, without destroying your git :D

Do i have to make a new branch and then add file, commit and push?

**Comment by velitasali** [comment, marked as answer]
Hi, I can help with no problem :)

There are multiple ways to send your changes, but to do it with GitHub, you can do it like this:

1- Fork this repo using the `Fork` button next to the `Star` button
2- Clone the forked repo
3- Create a copy of `it.po` (which is your translation work) inside the `po/` folder
4- Open a new terminal
5- Run `cd gnome-shell-extension-awesome-tiles`  (gnome-shell-extension-awesome-tiles being the cloned folder)
6- Run `git add po/it.po` which will stage your changes to be committed
7- Run `git commit -m 'i18n: Add Italian translation'` which will add a new commit with your changes titled *i18n: Add Italian translation*
8- Run `git push` which will send your changes to GitHub (it may ask username and password which is your GitHub login)
9- Go to https://github.com/velitasali/gnome-shell-extension-awesome-tiles/pulls
10- Click `New pull request`
11- Click your repo and its branch for the base
12- Click `Create pull request`
13- Click `Create pull request` again


I think this should do. If it seems too complicated, send me the po file and I can commit it for you :)

**Reply by Dolland** [reply]
Ah ok, i though it was enough to make a new branch.
so i have to make a new fork and then push changes, let me try :)
